style-transfer/transferStyle.js

```diff
@@ -0,0 +1,79 @@
+import {StyleTransfer} from './styleTransfer.js';
+
+// Operate Style Transfer Model
+export class TransferStyle {
```

Comment:
I am not quite convinced why we need this file. It looks like a thin wrapper to `FastStyleTransferNet`. It would also be good to have the pre-processing code (the `getInputTensor`, probably rename it to `preprocess`) besides the network code in `FastStyleTransferNet`. Could you please combine the two?